### PR TITLE
add billing cycle

### PR DESCRIPTION
**Pull request by neo773**
Screenshot 2023-09-08 at 9 13 18 PM

/claim #337

### CODE REVIEW

File: src/pages/dashboard/settings/Plans.vue [hunk 1]
```diff
@@ -146,6 +147,32 @@ const hightLights = computed<Stat[]>(() => ([
 
 <template>
   <div v-if="!isLoading" class="h-full bg-white max-h-fit dark:bg-gray-800">
+
+      <div class="px-4 pt-6 mx-auto max-w-7xl lg:px-8 sm:px-6" v-if="!displayPlans && currentPlan?.name !== 'free'">
```
Comment (rishi-raj-jain) [inline src/pages/dashboard/settings/Plans.vue]: Why the check for displayPlans?

File: src/pages/dashboard/settings/Plans.vue [hunk 2]
```diff
@@ -146,6 +147,32 @@ const hightLights = computed<Stat[]>(() => ([
...
+          <h1 class="text-5xl font-extrabold text-gray-900 sm:text-center dark:text-white">
+            {{ t('billing') }}
+          </h1>
+          <div   class="flex justify-center my-8" >
+
+              <div class="bg-gray-100 p-4 rounded-lg shadow-md w-[500px]">
+                <h2 class="text-2xl font-bold text-gray-900">Current Billing Cycle</h2>
+                <div class="flex justify-between mt-2">
+                  <div>
+                    <h3 class="text-lg font-semibold text-gray-700">From</h3>
```
Comment (rishi-raj-jain) [inline src/pages/dashboard/settings/Plans.vue]: Better choice of words is `Started On` and `Expires On`?

File: src/pages/dashboard/settings/Plans.vue (hunk 1 above)
Comment (WcaleNieWolny) [reply src/pages/dashboard/settings/Plans.vue]: Yeah, this change (`!displayPlans`) prevents this PR from working. Without it, it works great, but with it I cannot get this new div to show

File: src/pages/dashboard/settings/Plans.vue [hunk 3]
```diff
@@ -146,6 +147,32 @@ const hightLights = computed<Stat[]>(() => ([
...
+
+      <div class="px-4 pt-6 mx-auto max-w-7xl lg:px-8 sm:px-6" v-if="!displayPlans && currentPlan?.name !== 'free'">
+        <div class="sm:align-center sm:flex sm:flex-col">
+          <h1 class="text-5xl font-extrabold text-gray-900 sm:text-center dark:text-white">
+            {{ t('billing') }}
+          </h1>
+          <div   class="flex justify-center my-8" >
+
+              <div class="bg-gray-100 p-4 rounded-lg shadow-md w-[500px]">
+                <h2 class="text-2xl font-bold text-gray-900">Current Billing Cycle</h2>
```
Comment (WcaleNieWolny) [inline src/pages/dashboard/settings/Plans.vue]: Missing i18n

**Review by WcaleNieWolny — CHANGES_REQUESTED** [review]
Please lint this code with`pnpm lint --fix` and resolve issues mentioned bellow

**Comment by WcaleNieWolny** [comment]
Also, I know this is my subjective option but wouldn't centering the `Current Billing Cycle` look better?

Every element is centered except this text
image
image

A simple `text-center` could do the job
image

Let me know what do you think

**Review by WcaleNieWolny — APPROVED** [review]
LGTM

**Review by rishi-raj-jain — APPROVED** [review]

**Comment by rishi-raj-jain** [comment]
Nice!

**Pull request merged**